tutorials/getting-started-with-flow/01__how-to-use-vaadin-flow/content.adoc

```diff
@@ -19,7 +20,7 @@ The result is a single web page, consisting of a table (called a Grid in Vaadin)
 
 //[[figure.flow.tutorial.final-ui]]
 //.UI to be built in the tutorial
-image:FinishedUI.png[UI to be built in the tutorial]
+image:images/FinishedUI.png[UI to be built in the tutorial]
```

Comment:
I guess the images/ part is not needed. At least the images work for me without it